Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -747,6 +756,8 @@ private async void QueryResults()
             if (currentCancellationToken.IsCancellationRequested)
                 return;
 
+            // Update the query's IsForced property to true if this is a re-query
+            query.IsForced = reQuery;
```

Comment:
Suggestion to rename to `IsReQuery` from `IsForced`. You used `reQuery` everywhere, except for the flag name, both in the code and in descriptions and comments.

Reply:
yeah point taken, I myself was on the fence about how to name it ;)